docs/python/modelspec/programmatic/ste.rst

```diff
@@ -54,12 +54,12 @@ StateTransitionEvent
       
       h.steps_per_ms = 64
       h.dt = 1.0/h.steps_per_ms
-      print("dt=1/64 fixed step run %g" % h.dt)
+      print(f"dt=1/64 fixed step run {h.dt}")
       h.run()
 
       for i in [1,2]:
         h.cvode.condition_order(i)
-        print("cvode.condition_order() = %d" % int(h.cvode.condition_order()))
+        print(f"cvode.condition_order() = {h.cvode.condition_order()}")
```

Comment:
Note: `h.cvode.condition_order()` has returned an integer for many versions of NEURON now; there is no longer a need to cast its return.